src/gc/impl/conservative/gc.d

```diff
@@ -260,6 +294,7 @@ class ConservativeGC : GC
     import core.internal.spinlock;
     static gcLock = shared(AlignedSpinLock)(SpinLock.Contention.lengthy);
     static bool _inFinalizer;
+    static bool isPrecise=false;
```

Comment:
> I'm a little fuzzy on the proper usage of __gshared. Should anything marked __gshared be in a module's global scope or can they still be a member of a struct/class?

They can stay members, but you should remove `static`, as it might revert the effect of `__gshared`.